### ISSUE TITLE
bug in detection,model.eval() needs to be added

### ISSUE THREAD

**Issue by YFCYFC** [muvanpersie/yolo-fastest-pytorch#2]
Hi, this is an easy and amazing project, which is the most understandable yolo-fastest implemetation I found.When I finished training my own yolo-fastest model and detected some objects in some pictures, the results are not NORMAL, and I checked my data and training process, but no mistake found.Finally, I doubt that in detect.py, the state of model needs to be set as eval instead of doing nothing.And after doing that, the results are desirable.The solution is so simple that one line command "model.eval()" added is sufficient to product the good result.
I note that you set torch.no_grad() when conducting detection,but I believe it is important to make it explicit that torch.no_grad() and model.eval() have different effects on model.In inference phase, model.eval() is needed to tell model that some layers(for example dropout and batchnorm layers),which act differently in inference phase and training phase, should act as they are in inference phase.

Best wishes.

**Comment by muvanpersie** [comment]
Thanks so much for your suggestion. These days I concentrated on my work and I will check it when I am free. Thanks again!
